_config.yml

```diff
@@ -50,8 +61,8 @@ navigation:
 #      icon: people
   - title: Development
     pages:
-    - title: Quickstart
-      url: https://github.com/xenia-project/xenia#quickstart
+    - title: Building
+      url: https://github.com/xenia-project/xenia/blob/master/docs/building.md
```

Comment:
Why is Building replacing Quickstart here?

Reply:
Quickstart was made into a guide on how to use Xenia and was a duplicate of building.md